src/common/types.cpp

```diff
@@ -1274,6 +1271,7 @@ static idx_t GetLogicalTypeScore(const LogicalType &type) {
 		return 23;
 	// date/time/timestamp
 	case LogicalTypeId::TIME:
+	case LogicalTypeId::TIME_NS:
```

Comment:
IDK if it matters, but this doesn't follow the `TIMESTAMP` vs `TIMESTAPM_NS` below (it's 54 vs 55 there)